astrokat/simulate.py

```diff
@@ -287,27 +288,43 @@ def scan(
     def slew_time(self, target):
         """Get slew time.
 
-        How long in seconds, it took to the antennas to move
-        from previous target to the next
+        How long in seconds, it took for the antennas to move
+        from current target to the next.
+
+        Notes:
+            The slew time is calculated with consideration of az-el motion
+            of antennas instead of the current angular distance between sources,
+            assuming a slew_speed 2 deg/s. Antennas slew at 2 deg/s in az while
+            moving at 1 deg/s in el. There is about 3 seconds for overhead
```

Comment:
Is the `SIM_OVERHEAD` and this 3 seconds due to two different effects?  I'm wondering why we need to split the delays into two different places.

Reply:
Yes they are from different effects:

- `SIM_OVERHEAD` should be named correctly, it is the time between the log line:
2019-10-17 02:07:20Z - Initialising Track gaincal J0240-2309 for 120.0 sec
And this log line
`2019-10-17 02:07:XXZ - Slewing to ...`   # This has been removed in simulations
2019-10-17 02:07:42Z - Slewed to J0240-2309

Simulation log looks like this for now:

2019-10-17 02:07:20Z - Initialising Track gaincal J0240-2309 for 120.0 sec
2019-10-17 02:07:42Z - Slewed to J0240-2309